cms/templates/js/video-transcripts.underscore

```diff
@@ -0,0 +1,49 @@
+<div class='show-video-transcripts-container'>
+<% if (_.size(transcripts)) { %>
+    <button class="button-link toggle-show-transcripts-button">
+        <strong>
+        <span class="icon fa fa-caret-right toggle-show-transcripts-icon" aria-hidden="true"></span>
+        <span class="toggle-show-transcripts-button-text">
+            <%- StringUtils.interpolate(gettext('Show transcripts ({totalTranscripts})'), {totalTranscripts: _.size(transcripts)})%>
+        </span>
+        </strong>
+    </button>
+<% } else { %>
+    <span class='transcripts-empty-text'><%- gettext('No transcript available yet.') %></span>
```

Comment:
User facing string

Reply:
In progress

This is assuming that the system provides the transcripts (which is my understanding). If the user needs to add the transcript, I would change that to "No transcript uploaded." and possibly have a "Upload Transcript" option right under it.

Reply:
Great.

Button part is be the future work planned.
> Eventually there will be another button that will let user upload/add a new transcript (but that's future work).

Reply:
I will go for `No transcript uploaded.` option.